src/Tribe/Editor/REST/V1/Endpoints/Single_Ticket.php

```diff
@@ -231,13 +232,18 @@ public function add_ticket( WP_REST_Request $request, $nonce_action ) {
 			);
 		}
 
-		if ( $ticket_id === null && $provider_name !== null ) {
-			$provider = call_user_func( [ $provider_name, 'get_instance' ] );
-		} else {
+		if ( ! empty( $ticket_id ) ) {
 			$provider = tribe_tickets_get_ticket_provider( $ticket_id );
 		}
 
-		if ( ! $provider instanceof Tribe__Tickets__Tickets ) {
+		if (
+			empty( $provider )
+			&& ! empty( $provider_name )
+		) {
+			$provider = Tribe__Tickets__Tickets::get_ticket_provider_instance( $provider_name );
+		}
+
+		if ( empty( $provider)  ) {
```

Comment:
* :no_entry_sign: Expected exactly one space before closing parenthesis; "  " found.
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 59c689345178a9b21599bf40d255d470</sub>

Reply:
fixed